bootstrap-salt.sh

```diff
@@ -4185,6 +4194,43 @@ install_debian_check_services() {
 #   Fedora Install Functions
 #
 
+__install_saltstack_fedora_onedir_repository() {
+    if [ "$ITYPE" = "stable" ]; then
+        repo_rev="$ONEDIR_REV"
+    else
+        repo_rev="latest"
+    fi
+
+    __PY_VERSION_REPO="yum"
+    if [ -n "$_PY_EXE" ] && [ "$_PY_MAJOR_VERSION" -eq 3 ]; then
+        __PY_VERSION_REPO="py3"
+    fi
+
+    gpg_key="SALT-PROJECT-GPG-PUBKEY-2023.pub"
+
+    repo_file="/etc/yum.repos.d/salt.repo"
+
+    if [ ! -s "$repo_file" ] || [ "$_FORCE_OVERWRITE" -eq $BS_TRUE ]; then
+        fetch_url="${HTTP_VAL}://${_REPO_URL}/${_ONEDIR_DIR}/${__PY_VERSION_REPO}/fedora/${DISTRO_MAJOR_VERSION}/${CPU_ARCH_L}/${ONEDIR_REV}"
+        if [ "${ONEDIR_REV}" = "nightly" ] ; then
+            fetch_url="${HTTP_VAL}://${_REPO_URL}/${_ONEDIR_NIGHTLY_DIR}/${__PY_VERSION_REPO}/fedora/${DISTRO_MAJOR_VERSION}/${CPU_ARCH_L}/"
```

Comment:
This will still be broken as we  will now have at least two sub paths to account to, `master` and `<release-branch>`(3006.x).
And then we'll also have a date, today's date.
We should probably have a `latest` along with the dates, so it's simpler for scripts to get latest nightly.....
Anyway, this is still being worked on, so, while it's ok to commit this now, it will be broken soon.

Reply:
@s0undt3ch This brokeness is why I disabled all the nightly tests for now.

Reply:
👍